spec/genre_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './genre'
+
+describe Genre do
+  context 'when receiving valid parameters' do
+    let(:genre) { Genre.new('Rock') }
+    let(:item) { 'sample item' }
+
+    it 'returns a valid Genre object' do
+      expect(genre).to be_instance_of(Genre)
+    end
+
+    it 'add_item method updates the items instance variable' do
+      genre.add_item(item)
+      expect(genre.items).to eq([item])
+    end
+  end
```

Comment:
- [ ] One of the project requirements is that all public methods in all classes should have unit tests. Please write unit tests for the `to_hash` method in your `Genre` class 👍